src/ui/pages/work/activities/dialogs/AddActivityDialog.js

```diff
@@ -39,53 +38,55 @@ function AddActivityDialog(props) {
     };
 
     let handleSave = () => {
-        let activity = {...formValues}
+        let activity = { ...formValues };
         delete activity.taskDesc;
         delete activity.tradeDesc;
         delete activity.materialListDesc;
 
         setLoading(true);
         WSWorkorders.createWorkOrderActivity(activity)
-            .then(result => {
+            .then((result) => {
                 //Post add handler
                 props.postAddActivityHandler();
                 setLoading(false);
-                props.showNotification("Activity successfully created");
+                props.showNotification('Activity successfully created');
                 handleClose();
                 props.onChange();
             })
-            .catch(error => {
+            .catch((error) => {
                 setLoading(false);
-                props.handleError(error)
+                props.handleError(error);
             });
     };
 
     let updateFormValues = (key, value) => {
-        if (key === "taskCode") {
+        if (key === 'taskCode' && value) {
             onTaskCodeChanged(value);
         }
```

Comment:
If value is undefined, leads to a crash later on, stops crash on emptying field

Reply:
Sorry, for this error, it was when the backspace was pressed to remove any value that might be in the field. Once empty, the page would crash. This may have been fixed in a more recent push